src/GovUk.Education.ExploreEducationStatistics.Common/Extensions/DateTimeExtensions.cs

```diff
@@ -1,12 +1,23 @@
 ﻿using System;
+using System.Runtime.InteropServices;
 
 namespace GovUk.Education.ExploreEducationStatistics.Common.Extensions
 {
     public static class DateTimeExtensions
     {
-        public static DateTime AsMidnightLocalTime(this DateTime dateTime)
+        /**
+         * The resulting value corresponds to this DateTime with the time-of-day part set to zero (midnight) GMT/BST as a UTC date
+         */
+        public static DateTime AsMidnightConvertedToUtc(this DateTime dateTime)
```

Comment:
Think `AsStartOfDayUtc` would be less ambiguous. Midnight can be a bit ambiguous.

Reply:
I've renamed this now, thanks.